packages/design-system/src/components/OcResource/OcResource.vue

```diff
@@ -229,6 +242,32 @@ export default defineComponent({
     max-height: $oc-size-icon-default * 1.5;
     width: $oc-size-icon-default * 1.5;
     max-width: $oc-size-icon-default * 1.5;
+
+    &-status-badge {
+      position: absolute;
+      bottom: 0px;
+      right: 0px;
+      width: 8px;
+      height: 8px;
+      padding: var(--oc-space-xsmall);
+      line-height: var(--oc-space-small);
+      border-radius: 30px;
+      background: rgba(155, 155, 155, 0.8);
+      color: white;
+
+      .oc-icon {
+        margin-top: -2px;
+        margin-left: -1.5px;
+        svg {
+          fill: var(--oc-color-background-default) !important;
+        }
+      }
+      .oc-spinner {
+        margin-left: -2px;
+        margin-top: -2px;
+        color: var(--oc-color-background-default) !important;
+      }
```

Comment:
Is this needed? I don't see any occurrence of a spinner in this template.

Reply:
yes jan wants to use a spinner there as soon as its merged, its not needed in this PR theoretically but I wanted to make sure jan can just add a spinner without having to style it.

Reply:
The idea is great, but I think that's not the component's responsibility. That can land in our ResourceTable

Reply:
i will move it ^^, thx